course/templates/course/_course_menu.html

```diff
@@ -169,7 +169,7 @@ <h4>{% trans "Course staff" %}</h4>
 {% for group in course_menu.staff_link_groups %}
 {% if group.label %}
 <li role="presentation" class="header">
-	<h4>{{ group.label|parse_localization }}</h4>
+	<h4 id="course-menu-heading">{{ group.label|parse_localization }}</h4>
```

Comment:
Since this is in a for-loop, is it possible that this id would appear several times on the resulting page?

Reply:
This was added my mistake, thanks for spotting!